torch/quantization/fx/quantization_patterns.py

```diff
@@ -507,6 +510,25 @@ def convert(self, quantizer, node):
             quantizer.quantized_graph,
             node, quantizer.activation_post_process_map[node.name])
 
+class CustomModule(QuantizeHandler):
+    def convert(self, quantizer, node, load_arg, debug=False):
+        assert node.op == 'call_module'
+        observed_custom_module = quantizer.modules[node.target]
+        if node.name in quantizer.activation_post_process_map:
+            observed_custom_module.activation_post_process = \
+                quantizer.activation_post_process_map[node.name]
+        quantized_custom_module_class = \
+            get_quantized_custom_module_class(observed_custom_module._FLOAT_MODULE)
+        quantized_custom_module = \
+            quantized_custom_module_class.from_observed(observed_custom_module)
```

Comment:
The code for nn modules only has a from_float that gets called at convert time, this is a better name choice as we call "from_observed", but wondering if we can do anything to reduce inconsistency in names. Any ideas?

Reply:
here we are converting from observed to quantized, are you suggesting to keep the from_float name?

Reply:
No, this is a better name. Not sure what we can do for existing nn modules where we use from_float for both observers, marking as resolved

Reply:
i see, maybe we can refactor current from_float to from_observed as well